activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/resources/schema/process-extensions-schema.json

```diff
@@ -149,7 +164,10 @@
           "then": {
             "properties": {
               "value": {
-                "type": "boolean"
+                "type": "boolean",
+                "message": {
+                  "type": "mismatch type {{type}} - {{name}} {{id}}. expected type is boolean"
```

Comment:
Some messages for type mismatch contains `{{type}} - {{name}} {{id}` (i.e. `boolean`, `json`, `file`); others contains only `{{name}} {{id}`. Is there any specific reason for that?

Reply:
I forgot to update here. solved now. thanks